src/api/spec/models/package_spec.rb

```diff
@@ -20,48 +20,6 @@
   let(:other_user) { create(:confirmed_user, login: 'other_user') }
   let(:other_user2) { create(:confirmed_user, login: 'other_user2') }
   let(:other_user3) { create(:confirmed_user, login: 'other_user3') }
```

Comment:
Please add a note to this commit that tells why this test got (re)moved:-)

Reply:
done